src/Vulkan/Extensions/Silk.NET.Vulkan.Extensions.ImGui/imgui_impl_vulkan.cs

```diff
@@ -0,0 +1,1453 @@
+//////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// This is a C# port of the Vulkan imgui backend implementation:
+// https://github.com/ocornut/imgui/blob/master/backends/imgui_impl_vulkan.h
+// https://github.com/ocornut/imgui/blob/master/backends/imgui_impl_vulkan.cpp
+// (latest commit: https://github.com/ocornut/imgui/commit/6d3a980f38d849e9eac22180dce12f4c2e16794c)
+//
+// The names, layout, and overall convention was preserved as much as possible (almost), in order to make
+// future updates easy to apply.
+// Exceptions: Some unused methods were removed, as well as unused members of ImGui_ImplVulkan_InitInfo.
```

Comment:
Would rather align this with the rest of the codebase, with C#-style names and everything else.